charts/uplink/templates/tests/secret_test.yaml

```
@@ -0,0 +1,40 @@
+suite: test secret.yaml
+templates:
+  - secret.yaml
+tests:
+  - it: should ensure the secret has correct metadata
+    set:
+      postgresql.url: "example-url"
+      client.id: "example-id"
+      client.secret: "example-secret"
+      sentry.dsn: "example-dsn"
+    asserts:
+      - isKind:
+          of: Secret
+      - matchRegex:
+          path: metadata.name
+          pattern: "^uplink.*" # Assumes the "uplink.fullname" template results in names starting with "uplink"
```

```suggestion
      - equal:
          path: metadata.name
          value: "RELEASE-NAME-uplink."
```

We want to verify the exact behavior of the chart so a regex is not a good enough validation.
If you want to verify the naming, you have to chek the https://github.com/helm-unittest/helm-unittest documentation to see how it behaves (or look on what is the result of your test if it fails)